README.md

```diff
@@ -69,6 +69,20 @@ Before running tangle-accelerator, please edit binding address/port of accelerat
 $ make && bazel run //accelerator
 ```
 
+### Build from docker
+
+If you prefer building a docker image, tangle-accelerator also provides build rules for it. Note that you still have to edit configurations in `accelerator/config.h`.
```

Comment:
In preparation for image deployment, we might require runtime configurations for `Tangle-Accelerator`, otherwise the generated executable image could not really apply to various environments.

Reply:
I'm considering some kind of configuration file(yml maybe), so docker can also mount the volume.

Reply:
Please send another pull request and wait for its review.

Reply:
The approaches to configure Redis might be crucial, and we need to provide relevant advice.